packages/builder/src/components/integration/RestQueryViewer.svelte

```diff
@@ -50,18 +50,20 @@
     toBindingsArray,
   } from "@/dataBinding"
   import ConnectedQueryScreens from "./ConnectedQueryScreens.svelte"
+  import _ from "lodash"
```

Comment:
Not sure if the bundler would treeshake this or not, but we typically just import the function we need rather than the entirely of lodash. I doubt it actually makes a difference (and there's probably somewhere else in the codebase we import the whole thing like this), but we may as well just import what we need in case it does.

Reply:
It probably work fine, but it might be better to play safe. And I realised we were already importing it :P